ibft2-eks/README.md

```diff
@@ -0,0 +1,140 @@
+
+The Kubernetes metrics server is an aggregator of resource usage data in your cluster, and it is not deployed by default in Amazon EKS clusters. The Kubernetes dashboard uses the metrics server to gather metrics for your cluster, such as CPU and memory usage over time.
+
+Install metrics
+
+```bash
+DOWNLOAD_URL=$(curl -Ls "https://api.github.com/repos/kubernetes-sigs/metrics-server/releases/latest" | jq -r .tarball_url)
+DOWNLOAD_VERSION=$(grep -o '[^/v]*$' <<< $DOWNLOAD_URL)
+curl -Ls $DOWNLOAD_URL -o metrics-server-$DOWNLOAD_VERSION.tar.gz
+mkdir metrics-server-$DOWNLOAD_VERSION
+tar -xzf metrics-server-$DOWNLOAD_VERSION.tar.gz --directory metrics-server-$DOWNLOAD_VERSION --strip-components 1
+KUBECONFIG=$KUBECONFIG:/home/<username>/workspace/besu-aws/ibft2-k8s/.kube/config kubectl apply -f metrics-server-$DOWNLOAD_VERSION/deploy/1.8+/
+```
+
+Deploy the dashboard
+
+```bash
+KUBECONFIG=$KUBECONFIG:/home/<username>/workspace/besu-aws/ibft2-k8s/.kube/config kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta8/aio/deploy/recommended.yaml
+```
+
+Update permissions: By default, the Kubernetes dashboard user has limited permissions. Create an eks-admin service account and cluster role binding that you can use to securely connect to the dashboard with admin-level permissions
+```
+KUBECONFIG=$KUBECONFIG:/home/<username>/workspace/besu-aws/ibft2-k8s/.kube/config kubectl apply -f eks-admin-sa.yml
+```
+
+Retrieve an authentication token for the eks-admin service account. Copy the <authentication_token> value from the output. Watch for new lines in the token when pasting from terminal, hence we pipe to file
+```
+# get the exact name of the secret
+EKS_ADMIN_SECRET_NAME=`KUBECONFIG=$KUBECONFIG:/home/<username>/workspace/besu-aws/ibft2-k8s/.kube/config kubectl -n kube-system get secrets | grep 'eks-admin' | awk '{print $1}'`
+# get the value of the secret -
+KUBECONFIG=$KUBECONFIG:/home/<username>/workspace/besu-aws/ibft2-k8s/.kube/config kubectl -n kube-system describe secret $EKS_ADMIN_SECRET_NAME > /tmp/token.txt
+```
+Copy the token value from /tmp/token.txt`
+
+Now start the proxy
+```
+KUBECONFIG=$KUBECONFIG:/home/<username>/workspace/besu-aws/ibft2-k8s/.kube/config kubectl proxy &
+```
+
+Open the dashboard endpoint in a web browser: http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/#!/login.
+and choose **Token**, paste the <authentication_token> output from the `/tmp/token.txt`  into the **Token** field, and choose **SIGN IN**
+
+Now you should be able to get a lot of visibility into the cluster via the dashboard GUI
+
+5. Deploy the Besu chart
+
+Clone the example [besu kubernetes repo](https://github.com/PegaSysEng/besu-kubernetes) and select one of the helm deployments.
+
+Lets say 'ibft2' so to deploy the chart:
+
+```
+cd helm\ibft2
+KUBECONFIG=$KUBECONFIG:/home/<username>/workspace/besu-aws/ibft2-k8s/.kube/config helm install besu ./besu
```

Comment:
can we move this path reference to somethinh like `/path/to/repo/.kube/config`

Reply:
and everywhere else we refer to the `/home/<username>/workspace/besu-aws/` path

Reply:
done